app/main/posts/views/post-view-data.html

```diff
@@ -62,10 +62,9 @@
             <post-card
                 ng-repeat="post in posts"
                 can-select="bulkActionsSelected"
-                post="post.id === selectedPostId ? selectedPost.post : post"
```

Comment:
Is there a reason why you removed this? It breaks the two-way-data-binding that is needed to update the post-card on the left correctly after editing. Don't need selectedPostId though, works with                 `post="selectedPost.post && post.id === selectedPost.post.id ? selectedPost.post : post"` :)

Reply:
I'll double check but I think it was because the post was being always overwritten. I'll readd a connection to update the local post!

Reply:
Ah, it's because it was considering it to still be the selected post even when another Post becomes selected, in terms of the applying classes and when a new post moves to edit mode, as in overwrites the current selected post the old post is reverted.

This line applies the SelectedPost to the Post card if the selectedPost === to the Post's id, when that is no longer true it reverts to the original Post state.